package.json

```diff
@@ -366,6 +366,14 @@
       {
         "command": "cmake.outline.revealInCMakeLists",
         "title": "%cmake-tools.command.cmake.outline.revealInCMakeLists.title%"
+      },
+      {
+        "command": "cmake.hideLaunchCommand",
```

Comment:
You don't need to add this to `package.json`.  Just register it in `extension.ts`.  You extension can call it without it being in package.json.

Reply:
I got an error message when debugging the extension... Maybe I don't need to add anything to package.json?

Reply:
>Maybe I don't need to add anything to package.json?

That was my understanding.  For cpptools, commands that we don't want to show up in the command palette are not added to package.json.

Example:
```
    disposables.push(vscode.commands.registerCommand('cpptools.activeConfigName', onGetActiveConfigName));
```